sample/static-module/package.json

```diff
@@ -0,0 +1,33 @@
+{
+    "name": "static-module",
+    "author": "Workleap",
+    "version": "0.0.0",
+    "description": "Static module to showcase @squide.",
+    "license": "Apache-2.0",
+    "type": "module",
+    "main": "./dist/index.js",
+    "scripts": {
+        "dev": "tsup --config ./tsup.dev.ts"
+    },
+    "peerDependencies": {
+        "react": "*",
+        "react-dom": "*",
+        "react-router-dom": "*"
+    },
+    "devDependencies": {
+        "@types/react": "18.2.0",
+        "@types/react-dom": "18.2.0",
+        "@workleap/eslint-plugin": "1.2.0",
+        "@workleap/typescript-configs": "2.1.0",
+        "react": "18.2.0",
```

Comment:
react/react-dom/react-router-dom/react-router should be in dependencies, not build dependencies

Reply:
I just saw that you put them in peer dependencies. it works, but its odd for an app. I would put them in dependencies